Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pytest = "*"
+
+[packages]
+requests = "*"
+bcrypt = "*"
+python-gnupg = "*"
+pyopenssl = "*"
```

Comment:
This is for specifying versions? Can you pin major.minor versions? In order to make sure there are no compatibility issues, and that we can benefit from security updates?